sentry_streams/sentry_streams/pipeline/msg_parser.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from sentry_streams.pipeline.message import Message
```

Comment:
Unit tests please ensuring it works for protobuf as well

Reply:
You'll need a protobuf message type published to do that. If you need a hand with sentry-protos let me know.

Reply:
I'll come back to this after PTO in another PR. Thanks!